src/encoded/tests/features/search.feature

```diff
@@ -80,3 +80,8 @@ Feature: Search
         And I wait for the content to load
         When I fill in "searchOrgan" with "zzz"
         Then I should see 0 elements with the css selector "div.term-list.searchOrgan > li"
+
+    Scenario: Smoke testing advanced query search
+        When I fill in "searchTerm" with "@type:Experiment  date_created:[2015-01-01 TO 2018-12-31]"
```

Comment:
wait this should fail?  searchTerm -> advancedQuery?

Reply:
No. On the UI (HTML), the name-attribute on the search element is "searchTerm". So, it does not fail. I did not update the name though as it is beyond the scope of this ticket. On a different topic,  Emma brought up something with advancedQuery, dates and "clear filters" that I need to think about.